lib/ansible/modules/cloud/misc/proxmox.py

```diff
@@ -175,6 +175,14 @@
      - Indicate desired state of the instance
     choices: ['present', 'started', 'absent', 'stopped', 'restarted']
     default: present
+  pubkey:
+    description:
+      - Public key to add to /root/.ssh/authorized_keys.
```

Comment:
If I'm reading the code correct, does this only work on proxmox 4.2+? If so may be worth stating that here.